cmd/notify-mailer/main.go

```diff
@@ -362,20 +427,23 @@ func main() {
 	// Load email body
 	body, err := ioutil.ReadFile(*bodyFile)
 	cmd.FailOnError(err, fmt.Sprintf("Reading %q", *bodyFile))
+	template, err := template.New("email").Parse(string(body))
+	cmd.FailOnError(err, "Parsing template")
 
 	address, err := mail.ParseAddress(*from)
 	cmd.FailOnError(err, fmt.Sprintf("Parsing %q", *from))
 
-	toBody, err := ioutil.ReadFile(*toFile)
-	cmd.FailOnError(err, fmt.Sprintf("Reading %q", *toFile))
+	recipients, err := readRecipientsList(*recipientListFile)
+	cmd.FailOnError(err, fmt.Sprintf("Reading %q", *recipientListFile))
```

Comment:
Can you also add a check that fails early if `len(recipients) == 0`?

Reply:
I wound up putting this check inside `readRecipientsList` so it can be caught by this `FailOnError`.